src/patternfly/components/Dropdown/docs/code.md

```diff
@@ -38,7 +38,7 @@ The dropdown menu can contain either links or buttons, depending on the expected
 | `.pf-m-align-right` | `.pf-c-dropdown__menu` | Modifies to display the menu aligned to the right edge of the toggle |
```

Comment:
in these docs, there is a line for

> aria-expanded=“true” | .pf-c-dropdown__menu | ...

I don’t think we need that? `aria-expanded` goes on the toggle, not the menu.

Reply:
yeah this was also on the app launcher component so removed them both.

Reply:
Can you also remove this? applied to the select dropdown which is it's own component now.

> aria-pressed="true" | button.pf-c-dropdown__menu-item | Select only The attribute aria-pressed="true" should be set programmatically to the active item. Value should be false when not selected.
-- | -- | --